### DISCUSSION TITLE
About CVE-2023-4863 libwebp vulnerability

### DISCUSSION THREAD

**Discussion by SagePtr** [oerdnj/deb.sury.org#2026] [Q&A] [answered]
Heap buffer overflow in libwebp.
Fixed in latest Debian: https://security-tracker.debian.org/tracker/CVE-2023-4863
Fixed in latest Ubuntu: https://ubuntu.com/security/CVE-2023-4863
At least, php_gd2 uses libwebp, not sure about php_imagick.
Shouldn't PHP be recompiled against patched libwebp?

**Comment by oerdnj** [comment, marked as answer]
No, libwebp is dynamically linked, so there's no need to recompile anything.  However you should restart any process that might have pulled the old vulnerable version into its address space.
